types/tools.ts

```
@@ -2,7 +2,7 @@ import type { Tool } from "@/db/schema/tools";
 
 export type ToolFormatted = Omit<
   Tool,
-  "authenticationToken" | "authenticationMethod" | "createdAt" | "updatedAt" | "headers"
+  "authenticationToken" | "authenticationTokenPlaintext" | "authenticationMethod" | "createdAt" | "updatedAt" | "headers"
```

ESLint

[error] Replace `"authenticationToken"·|·"authenticationTokenPlaintext"·|·"authenticationMethod"·|·"createdAt"·|·"updatedAt"` with `|·"authenticationToken"⏎··|·"authenticationTokenPlaintext"⏎··|·"authenticationMethod"⏎··|·"createdAt"⏎··|·"updatedAt"⏎·`

(prettier/prettier)